src/seals/atari.py

```diff
@@ -0,0 +1,1245 @@
+"""Adaptation of Atari environments for specification learning algorithms."""
```

Comment:
1.2k line file holy shit! Can't we do this programatically?

We don't need a separate function for each, `gym.register` has a `kwargs` parameter that gets passed into `EnvSpec` and then passed into the entry point (as keywor arguments, shockingly) when you call `gym.make`:
https://github.com/openai/gym/blob/master/gym/envs/registration.py#L610

So I think this file can be replaced by a single function:

```python
def fixed_length_atari(atari_env_id):
    """Fixed-length variant ..."""
    return AutoResetWrapper(gym.make(atari_env_id))
```

and then in call to `gym.register`, just include `kwargs=dict(atari_env_id=gym_name)`.

Reply:
Separately calling `gym.make` also feels a bit off to me. If we call `gym.make(seals/SomeAtariEnv-v4)` we're calling `gym.make` *again* internally and probably getting stuff you don't want (like an extra time limit wrapper).

I think the option I favor is to actually pass the `env_spec` in directly, extract the entry point and keyword arguments, then call that without calling `gym.make`. I don't feel that strongly about this though.

Reply:
I also found this to be extremely long, but figured that was the necessary way gym worked.

Reply:
So are you suggesting doing `gym.register(id=seals_name, entry_point="seals.atari.fixed_length_atari", kwargs=dict(atari_env_id=gym_name))`?

Reply:
Now we're at DRYing the code, is there a way to automatically extract the list of available environments without having to have a massive constant list `ATARI_ENV_NAMES`?

Reply:
Yes pretty much that.

I already suggested a way to replace `ATARI_ENV_NAMES`

Reply:
Good thing I replaced this, apparently I missed a bunch of environments located in https://github.com/mgbellemare/Arcade-Learning-Environment/blob/master/src/games/Roms.cpp